plugins/vli/vli-pd.quirk

```diff
@@ -71,6 +71,16 @@ GType = FuVliPdDevice
 Plugin = vli
 GType = FuVliPdDevice
 
+# Lenovo Mini Dock
+[DeviceInstanceId=USB\VID_17EF&PID_3093]
+Plugin = vli
+GType = FuVliPdDevice
+
+# Lenovo USB-C to 4 USB-A Hub
+[DeviceInstanceId=USB\VID_17EF&PID_103B]
+Plugin = vli
+GType = FuVliPdDevice
+
```

Comment:
@memily this isn't correct surely, if they are shared PD? i.e. there's one physical SPI flash chip on the Lenovo USB-C to 4 USB-A Hub, right?